config/HH_bbWW/Run3_2022EE/samples.yaml

```diff
@@ -0,0 +1,640 @@
+############## Start Resonant Signal samples)##############
```

Comment:
Missing samples:
- VBFHHto2B2V_CV-m1p83_C2V-3p57_C3-m3p39
- VBFHHto2B2Vto2L2Nu_CV-m1p83_C2V-3p57_C3-m3p39

Missing generator:
- GluGlutoRadiontoHHto2B2Vto2B2JLNu_M-1800
- GluGlutoRadiontoHHto2B2Vto2B2JLNu_M-3000
- GluGlutoHHto2B2V_kl-1p00_kt-1p00_c2-0p00

Set cross-section for all samples (to allow running the code). You can use 1pb placeholder for nonres samples